.github/config.yml

```diff
@@ -15,17 +15,16 @@ newPRWelcomeComment: >
 # Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
 # Comment to be posted to on pull requests merged by a first time user
 firstPRMergeComment: >
-  Thanks for your contribution to Meshery! :tada:
+  Thanks for your contribution to the Meshery! :tada:
    
   <picture>
-  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/meshery/meshery-perf/master/.github/welcome/Meshery-celebration-Dark.svg">
-  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/meshery/meshery-perf/master/.github/welcome/Meshery-celebration-Light.svg">
-  <img alt="Meshery Logo" src="">
+  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/meshery/meshery-nighthawk/master/.github/welcome/meshery-darkmode-celebration.svg">
```

Comment:
I went ahead and opened https://github.com/meshery/meshery-nighthawk/issues/22 to bring in a bit more Nighthawk to the repo.